input/pagecontent/general-requirements.md

```diff
@@ -300,7 +130,7 @@ Example: AU Core AllergyIntolerance profile showing clinicalStatus and verificat
 
 #### Interpreting profile elements labelled Must Support
 
-Profiles defined in this implementation publication flag *Must Support* on elements and not part sub-elements of a data type. 
+Profiles defined in this implementation publication flag *Must Support* on elements (e.g. `Patient.name`) and part sub-elements of a data type (e.g. `Patient.name.use`). 
```

Comment:
I don't think the term "part sub-elements" is needed here and can be confusing. Can simply say "sub-elements of a data type (e.g. `Patient.name.use`)"

Reply:
fixed.